tests/smoke-tests/src/test/java/org/apache/activemq/artemis/tests/smoke/console/QueuesTest.java

```diff
@@ -181,7 +181,8 @@ private void testExpiryQueue(String queueName, String expiryQueueName) throws Ex
       sendMessagePage.appendMessageText("xxx");
       sendMessagePage.sendMessage();
 
-      QueuePage queuePage = sendMessagePage.getQueuesPage(DEFAULT_TIMEOUT).getQueuePage(queueName, DEFAULT_TIMEOUT);
+      beforeQueuesPage.refresh(DEFAULT_TIMEOUT);
```

Comment:
The current page is sendMessagePage
```suggestion
      sendMessagePage.refresh(DEFAULT_TIMEOUT);
```